kfs-core/src/main/java/edu/arizona/kfs/coa/document/OrgReviewRoleMaintainableImpl.java

```diff
@@ -0,0 +1,45 @@
+package edu.arizona.kfs.coa.document;
+
+import org.kuali.kfs.coa.identity.OrgReviewRole;
+import org.kuali.rice.kns.web.ui.Field;
+
+public class OrgReviewRoleMaintainableImpl extends org.kuali.kfs.coa.document.OrgReviewRoleMaintainableImpl {
+
```

Comment:
This code was not included in the original modification, but I think it was missed:
 protected void prepareFieldsCommon(Field field, boolean shouldReviewTypesFieldBeReadOnly, boolean hasAccountingOrganizationHierarchy){
...
if(OrgReviewRole.REVIEW_ROLES_INDICATOR_FIELD_NAME.equals(field.getPropertyName())) {
            field.setReadOnly(true);
        } else if( !hasAccountingOrganizationHierarchy
                && (OrgReviewRole.FROM_AMOUNT_FIELD_NAME.equals(field.getPropertyName()) ||
                        OrgReviewRole.TO_AMOUNT_FIELD_NAME.equals(field.getPropertyName()) ||
Add new fields here ---->
                        OrgReviewRole.OVERRIDE_CODE_FIELD_NAME.equals(field.getPropertyName()))) {
            field.setReadOnly(true);
        }
This is so that only documents that have the AccountingOrganizationHierarchy routing node can use the new fields. Please let me know if you want to discuss.